src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const baseCost = 40 * days;
```

Comment:
Consider replacing the magic number `40` with a named constant, such as `DAILY_RENTAL_COST`, to improve code readability and maintainability.